guide/case-studies/blixt-wallet.md

```diff
@@ -32,7 +32,7 @@ Illustration sources: https://www.figma.com/file/eyQDCaMoQvnRVcHzSsgrzs/Blixt-UX
 
 ## How do we help users new to lightning start using their wallet?
 
-The [Bitcoin Design Community]({{ site.slack_invite_url }}) ran a design sprint on an open-source lightning wallet.  During this sprint, we collaboratively focused on re-designing one aspect of the user flow, the **onboarding journey**. In this collaborative effort, [we got together](https://www.youtube.com/watch?v=nctibe_a4rM&list=PLpV0KfVOMojYywq1Blp3SGx62cSyoRfHt&index=3) as a group of designers and researchers,  and aimed to improve the user experience using good design principles, and incorporating as much feedback from users as possible.
+The [Bitcoin Design Community]({{ site.discord_invite_url }}) ran a design sprint on an open-source lightning wallet.  During this sprint, we collaboratively focused on re-designing one aspect of the user flow, the **onboarding journey**. In this collaborative effort, [we got together](https://www.youtube.com/watch?v=nctibe_a4rM&list=PLpV0KfVOMojYywq1Blp3SGx62cSyoRfHt&index=3) as a group of designers and researchers,  and aimed to improve the user experience using good design principles, and incorporating as much feedback from users as possible. 
```

Comment:
trailing space, the ultimate nit. only fix if you happen to make more changes

Reply:
Fixed, had to fix something else.